docs/release.md

```diff
@@ -0,0 +1,149 @@
+# steps
+
+1. Sync latest master
+1. new branch
+1. Sync the downstream pipeline repo
+1. Checkout release branch
+1. Copy the release yaml from the pipelines repo to operator
+   deploy/resources/<version>
+1. modify operator config_controller.go to update the <version>
+1.  test the operator using `up local`
+1. build image
+```
+operator-sdk build quay.io/openshift-pipeline/openshift-pipelines-operator:v0.7.0
+
+```
+
+1. update `deploy/operator.yaml` image:
+
+1. generate csv
+
+```
+operator-sdk olm-catalog gen-csv \
+  --csv-channel dev-preview \
+  --csv-version 0.7.0 \
+  --from-version 0.5.2 \
+  --operator-name  openshift-pipelines-operator \
+  --update-crds
+```
+
+You might  need  to edit the package.yaml to remove any duplicate channels
+
+e.g.
+```
+channels:
+- currentCSV: openshift-pipelines-operator.v0.5.0
+  name: dev-preview
+- currentCSV: openshift-pipelines-operator.v0.7.0
+  name: dev-preview
+```
+will need to be corrected to:
+
+```
+channels:
+- currentCSV: openshift-pipelines-operator.v0.7.0
+  name: dev-preview
+```
+
+See existing pacakge in community operators for reference
+
+
+1. flatten  to generate bundle
+```
+operator-courier flatten \
+  deploy/olm-catalog/openshift-pipelines-operator \
+  tmp/openshift-pipelines-operator
+```
+1. verify that
+```
+operator-courier verify --ui_validate_io \
+  tmp/openshift-pipelines-operator
+```
+
+1. operator-courier use that to push the app bundle
+
+NOTE:
+
+You can obtain quay token by running `./scripts/get-quay-token` in
```

Comment:
provide link to doc
https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#push-to-quayio